core/src/mindustry/world/blocks/defense/turrets/TractorBeamTurret.java

```diff
@@ -39,6 +35,11 @@
     public Sound shootSound = Sounds.tractorbeam;
     public float shootSoundVolume = 0.9f;
 
+    public @Load("block-@size") TextureRegion baseRegion;
+    public @Load("@-laser") TextureRegion laser;
+    public @Load(value = "@-laser-start", fallback = "@-laser-end") TextureRegion laserStart;
```

Comment:
What's the use case for this?

Reply:
![image](https://user-images.githubusercontent.com/54301439/119932267-c8543380-bf37-11eb-80c3-e9fed9c38be7.png)

Reply:
that's a very strange use case